FlowCrypt/src/main/java/com/flowcrypt/email/api/email/JavaEmailConstants.kt

```diff
@@ -91,5 +91,8 @@ class JavaEmailConstants {
 
     const val EMAIL_PROVIDER_GMAIL = "gmail.com"
     const val EMAIL_PROVIDER_GOOGLEMAIL = "googlemail.com"
+    const val EMAIL_PROVIDER_YAHOO = "yahoo.com"
+    const val EMAIL_PROVIDER_OUTLOOK = "outlook.com"
```

Comment:
I'd add more:
hotmail.com
yahoo.co.uk
yandex.com[ru, ua, kz, by]
mail.ru (and their subdomains like hotbox.ru, list.ru etc)
rambler.ru